Plugins/Public/playercntl_plugin/HyperJump.cpp

```diff
@@ -1798,56 +1507,25 @@ namespace HyperJump
 				PrintUserCmdText(iClientID, L"WARNING NO JUMP COORDINATES");			
 				pub::Player::SendNNMessage(iClientID, pub::GetNicknameId("nnv_jumpdrive_blind_jump_warning"));
 			}
-		////////////////////////// System limit restriction ///////////////////////////////////////
-		if (JumpSystemListEnabled == 1 && jd.iTargetSystem != 0)
-		{
-			const Universe::ISystem *iSys = Universe::get_system(iSystemID);
-		    wstring wscSysName = HkGetWStringFromIDS(iSys->strid_name);
-			//PrintUserCmdText(iClientID, L"Space kitteh knows you are in %u !", iSystemID);
-			for (map<uint, SYSTEM_JUMPLIST>::iterator iter = mapJumpSystems.begin(); iter!=mapJumpSystems.end(); iter++)
+			////////////////////////// System limit restriction ///////////////////////////////////////
+			else if (JumpSystemListEnabled == 1 && jd.iTargetSystem != 0)
```

Comment:
`jd.iTargetSystem != 0` is redundant in this case, this is an `else if` where the original `if` condition was `jd.iTargetSystem == 0`

Reply:
Also, checking a bool is equal to 1 seems strange, just directly use the boolean